src/base/c_utils.h

```diff
@@ -52,5 +59,19 @@ inline void GetLastError(String* error) {
   }
 }
 
+inline bool SetPermissions(const Path& path, const Perms permissions,
```

Comment:
It should fit the machine word, but I'll make it a const ref no difference for me.